docs/apache-airflow/core-concepts/params.rst

```diff
@@ -159,6 +159,11 @@ JSON Schema Validation
         },
     ):
 
+.. note::
+    Schema validation on DAG definitions is only made if a schedule is defined. If a manual or external trigger
+    is defined (e.g. via ``schedule=None``) then params are validated during submission. This allows to enforce
+    undefined parameters being defined at trigger time.
+
```

Comment:
```suggestion
.. note::
    Schema validation on DAG definitions is only made if ``schedule`` is provided. If ``schedule=None`` then params are validated during submission. This allows to enforce
    undefined parameters being defined at trigger time.

```
the sentence `This allows to enforce undefined parameters being defined at trigger time.` remains confusing to me

Reply:
Sorry, the not native speaker english... thanks for the feedback. Tried to re-phrase a bit - better now?